README.md

```diff
@@ -6,6 +6,6 @@ Model Evaluation Tools (MET) Repository
 [![Docs](https://img.shields.io/badge/Documentation-latest-brightgreen.svg)](https://met.readthedocs.io)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5565322.svg)](https://doi.org/10.5281/zenodo.5565322)
 
-This repository contains the source code for the Model Evaluation Tools package (met), unit test code (test), and scripts used to build and test the code (scripts).
+This repository contains the source code for the Model Evaluation Tools package.
```

Comment:
I see you took out the following text:

> (met), unit test code (test), and scripts used to build and test the code (scripts).

but doesn't the repo still contain the unit test code and scripts, etc.?

Reply:
Yes, it does. They now live in the "internal" directory. I just decided to provide less details in that README file. They were only explicitly called out because those were the top-level directories and I wanted to give a succinct description of them. Now that they've moved, there's less of a need to describe them here.

Do you think more detail is needed in this README file?

Reply:
Thanks for explaining.  I think less detail is totally fine and that change makes sense.  I agree with what you did and why.